charts/kserve-resources/templates/configmap.yaml

```diff
@@ -273,7 +274,7 @@ data:
            # Namespace of the inference service ( {{- "{{ .Namespace }}" -}} )
            # For more info https://github.com/kserve/kserve/issues/2257.
            # NOTE: This configuration only applicable to serverless deployment.
-           "pathTemplate": "/serving/{{ .Namespace }}/{{ .Name }}"
+           "pathTemplate": ""
```

Comment:
IMO this is just an example that shows how to use the placeholders, and can stay unchanged. The previous block where you added this field should be enough to show what is the default value.

Reply:
I think can keep it as example for reference.